manuscript.md

```diff
@@ -2,234 +2,111 @@
+Spatial biases in data coverage are
+prevalent at the global scale (with South America, Africa and Asia being underrepresented) and different interaction types show biases towards different biomes (or environmental conditions) [@Poisot2020EnvBia]. These 'spatial gaps' serve as a limitation to our ability to confidently make predictions when accounting for real-world environmental conditions, especially in environments for which there are no analogous data.
+
+### Methods
+This scarcity of data limits the range of computational tools than can be used by network ecologists. Most deep learning methods, for instance, are very data expensive.
+The paucity of data is compounded by a collection of biases that can be found in existing datasets. Species interaction datasets are typically dominated by food webs, pollination, and host-parasite networks [@Ings2009EcoNet; @Poisot2020EnvBia]. This could prove to be a limiting factor when trying to understand or predict networks of *under represented* interaction types or trying to integrate networks of different types [@Fontaine2011EcoEvo], especially given the structural variation of ecological networks [@Michalska-Smith2019TelEco]. This stresses the need for an integrated, flexible, and data-efficient set of computational tools which will allow us to predict ecological networks accurately from existing and imperfect datasets.  
+
+### Scale
+We are also currently limited by the the level of biological organisation at which we can describe ecological networks. For instance, our understanding of individual based networks [see for example @Araujo2008NetAna; @Tinker2012StrMec] is still in its infancy [@Guimaraes2020StrEco] and acts as a resolution-limit at which we would be able to predict networks. On the note of scale, the resolution of environmental (or landscape) data would also limit our ability to predict networks at finer scales, although current trends in e.g. remote sensing would suggest that with time this would become less of a hindrance [@Makiola2020KeyQue].
+
+Ecosystems are a quintessential complex-adaptive-system []. In figure @fig:everything_is_connected, we see the myriad of ways in which processes at different spatial, temporal, and organizational scales influence and respond to one another. 
+Understanding how the product of these different processes drive the properties of ecosystems especially at different scales remains difficult. Working on understanding how these different 'snapshots' are connected and integrating that into future workflows/methods.
+
+![](figures/everything_connected_v2.png){#fig:everything_is_connected}
+
+
+## What is paving the path towards understanding species interaction networks?
```

Comment:
I thought to retool the title as the previous one made it seem like this is the path where we are more discussing the enabling part of things still. But feel free to revert